t/document/file.t

```diff
@@ -212,6 +212,13 @@ subtest 'Packages starting with underscore are not indexed' => sub {
     is( $file->module->[0]->indexed, 0, 'Package is not indexed' );
 };
 
+subtest 'files listed under other files' => sub {
+    my $file = new_file_doc( name => 'Makefile.PL' );
+    $file->set_authorized( $file->module => $file->author );
```

Comment:
These arguments `( $file->module => $file->author )` are confusing as they don't match the sub definition.
They would probably throw an error except that your change returns from the sub before the arguments are used.
Please remove them to avoid someone looking at the code and then wondering the same thing that I just did.